assets-sync.js

```diff
@@ -1311,60 +1312,135 @@ class EntityMigration {
         const data = await Promise.all(promises);
         return str.replace(regex, () => data.shift());
     }
+
     async migrateEntity(type, data) {
         switch (type) {
             case 'Actor':
             case 'actors':
                 data.img = await this._migrateEntityPath(data.img);
-                if (data.token)
+
+                if (data.prototypeToken) {
+                    data.prototypeToken = await this.migrateEntity('tokens', data.prototypeToken);
+                } else if (data.token) {
                     data.token = await this.migrateEntity('tokens', data.token);
+                }
+
                 if (data.items)
                     data.items = await this.constructor.mapAsync(data.items, item => this.migrateEntity('items', item));
-                if (data.data && data.data.details && data.data.details.biography && data.data.details.biography.value)
+                if (data.effects)
+                    data.effects = await this.constructor.mapAsync(data.effects, effect => this.migrateEntity('effects', effect));
+
+                if (data.system?.details?.biography?.value) {
+                    data.system.details.biography.value = await this._migrateHTML(data.system.details.biography.value);
+                } else if (data.data?.details?.biography?.value) {
                     data.data.details.biography.value = await this._migrateHTML(data.data.details.biography.value);
+                }
+                break;
+            case 'Adventure':
+            case 'adventures':
+                data.img = await this._migrateEntityPath(data.img);
+                data.caption = await this._migrateHTML(data.caption);
+                data.description = await this._migrateHTML(data.description);
+                [
+                    'actors',
+                    'combats',
+                    'items',
+                    'scenes',
+                    'journal',
+                    'tables',
+                    'macros',
+                    'cards',
+                    'playlists',
+                    'folders',
+                ].forEach(type => data[type] = await this.migrateEntity(type, data));
```

Comment:
oh, I missed that this should be sending `data[type]` to `migrateEntity`, rather than `data`.